test/integration/query-interface/schemas.test.ts

```diff
@@ -36,4 +38,64 @@ describe('QueryInterface#createSchema', async () => {
 
     expect(queryGeneratorSpy.args[0]).to.include(options);
   });
+
+  it('drops a schema', async () => {
+    await queryInterface.createSchema(testSchema);
+    const preDeletionSchemas = await queryInterface.showAllSchemas();
+    expect(preDeletionSchemas).to.include(testSchema, 'createSchema did not create testSchema');
+
+    await queryInterface.dropSchema(testSchema);
+    const postDeletionSchemas = await queryInterface.showAllSchemas();
+    expect(postDeletionSchemas).to.not.include(testSchema, 'dropSchema did not drop testSchema');
+  });
+
+  it('shows all schemas', async () => {
+    await queryInterface.createSchema(testSchema);
+    const allSchemas = await queryInterface.showAllSchemas();
+
+    const expected = dialectsWithEqualDBsSchemas.includes(dialect.name)
+      ? [sequelize.config.database, testSchema]
+      : [testSchema];
+    expect(allSchemas.sort()).to.deep.eq(expected.sort());
+  });
+
+  describe('drops all schemas', () => {
+    it('drops all schemas except default schema', async () => {
+      await queryInterface.dropAllSchemas({
+        skip: [sequelize.config.database],
```

Comment:
`sequelize.config.database` is only the default schema in mysql/mariadb.

The test for `skip` should work on all dialects so I recommend creating a few dummy schemas and skipping one of them instead